src/mixins/currency-lookup-mixin.ts

```diff
@@ -0,0 +1,176 @@
+import { Component, Vue } from 'vue-property-decorator'
```

Comment:
Did you consider using a npm package to provide these (ie, something we don't have to maintain)?
For example, https://www.npmjs.com/package/currency-codes ?

Why implement this way instead?

Reply:
Yes. I also considered couple of other options. The order in which the currency needs to be displayed , the item text will still need a lot of data reordering and stuff. Also whether all  options to be shown in the dropdown is another open question. This will be looked into by business once this is deployed to dev. This question was also asked in the demo.

Reply:
OK, perfect, this conversation is resolved!